pyproject.toml

```
@@ -26,10 +26,10 @@ dependencies = [
 
 [dependency-groups]
 dev = [
-    "black>=25.1.0",
     "django-stubs>=5.1.3",
     "pytest-cratedb>=0.4.0",
     "requests>=2.32.3",
+    "ruff>=0.14.10",
```

It's better like this, to not cause any behaviour drift, due to new formatting rules, and deprecations of older ones.
```suggestion
    "ruff<0.15",
```

---

Dependabot also understands this paradigm, and will submit corresponding patches, so maintainers can either merge without ado (if nothing changes), or accompany the PR with required adjustments about code formatting rules coming from a newer minor version of ruff.

- https://github.com/crate/cratedb-toolkit/pull/535